server/validation/validateAddProject.js

```diff
@@ -0,0 +1,13 @@
+const yup = require('yup');
+
+const addProjectSchema = yup.object({
```

Comment:
We will use the same schema for add and edit so it's great to be projectSchema
```suggestion
const projectSchema = yup.object({
```

Reply:
in validateEditProject i add another argument (projectI) that come from params, but in this does not exist!